app/models/foreman_tasks/task/dynflow_task.rb

```diff
@@ -70,7 +76,27 @@ def cli_example
 
     def main_action
       return @main_action if @main_action
-      execution_plan.root_plan_step.action execution_plan
+      action = execution_plan_action
+      if activejob?
```

Comment:
Because of this the result is sometimes an `Action` and sometimes an `ActiveJob`. This can lead to errors in [here](https://github.com/dLobatog/foreman-tasks/blob/d82379fed94983d3c88144361ac2d20faba4b3c2/app/models/foreman_tasks/task/dynflow_task.rb#L18), where `ActiveJob` doesn't respond to `caller_execution_plan_id` and raises `NoMethodError`. I hit this several times today